robottests/test_remotescript.robot

```diff
@@ -19,7 +19,7 @@ ${TESTFILESIZE}=    1000000
 Remote Test Setup
     Better Set RemoteScript Targets
 
-Better Set RemoteScript Targets
+Better Set Sudo RemoteScript Targets
```

Comment:
What does 'Better' mean here? Could it be removed/enhanced? It is rather difficult to understand by the reader.